dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -58,6 +58,11 @@ def get_display_theme
     render json: {display_theme: @user&.display_theme}
   end
 
+  # GET /api/v1/users/<user_id>/mute_music
+  def get_mute_music
+    render json: {mute_music: !!@user.mute_music}
```

Comment:
nit: use `@user&`

Reply:
Thanks for this- I updated the using_text_mode instances too.